pkg/scheduler/plugins/coscheduling/plugin_service.go

```diff
@@ -0,0 +1,22 @@
+package coscheduling
+
+import (
+	"net/http"
+
+	"github.com/gin-gonic/gin"
+
+	"github.com/koordinator-sh/koordinator/pkg/scheduler/frameworkext/services"
+	"github.com/koordinator-sh/koordinator/pkg/scheduler/plugins/coscheduling/util"
+)
+
+var _ services.APIServiceProvider = &Coscheduling{}
+
+func (p *Coscheduling) RegisterEndpoints(group *gin.RouterGroup) {
+	group.GET("/gang/:namespace/:name", func(c *gin.Context) {
+		gangNamespace := c.Param("namespace")
+		gangName := c.Param("name")
+		gangId := util.GetId(gangNamespace, gangName)
+		gang := p.pgMgr.GetGangByGangId(gangId)
```

Comment:
if gang is nil, should return 404 with services.ResponseErrorMessage like the following:
```go
ResponseErrorMessage(c, http.StatusNotFound, "cannot find gang %s/%s", gangNamespace, gangName)
```

Reply:
got it